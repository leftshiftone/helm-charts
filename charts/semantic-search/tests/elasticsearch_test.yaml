suite: test elasticsearch
templates:
  - elasticsearch.yaml
tests:
  - it: default
    set:
      elasticsearch.config.node.store.allow_mmap: false
    asserts:
      - isKind:
          of: Elasticsearch
      - equal:
          path: metadata.name
          value: semantic-search-elasticsearch
      - matchSnapshot:
          path: spec
