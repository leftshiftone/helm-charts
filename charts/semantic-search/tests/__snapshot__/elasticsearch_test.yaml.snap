default:
  1: |
    nodeSets:
    - config:
        node:
          store:
            allow_mmap: false
      count: 3
      name: semantic-search
      volumeClaimTemplates:
      - metadata:
          name: elasticsearch-data
        spec:
          accessModes:
          - ReadWriteOnce
          resources:
            requests:
              storage: 10Gi
    version: 8.4.0
